README.adoc

```diff
@@ -49,13 +49,11 @@ each property is a field in a JSON object like this:
 }
 ----
 
-When you create a new REST application, the design of the API is important. The JAX-RS APIs can be
-used to create JSON-RPC, or XML-RPC APIs, but it wouldn't be a RESTful service. A good RESTful service
-is designed around the resources that are exposed, and on how to create, read, update, and delete the
-resources.
-
-The service responds to `GET` requests to the `/System/properties` path. The `GET` request should
-return a `200 OK` response that contains all of the JVM's system properties.
+The design of an HTTP API is essential when creating a web application. The REST API has exclusively 
```

Comment:
I would drop the word "exclusively".  Newer HTTP-based technologies, like GraphQL and gRPC are gaining in popularity.

Reply:
Addressed.